zebra/main.c

```diff
@@ -312,17 +314,48 @@ FRR_DAEMON_INFO(zebra, ZEBRA,
 );
 /* clang-format on */
 
+void zebra_main_router_started(void)
+{
+	/* After we have successfully acquired the pidfile, we can be sure
+	 * about being the only copy of zebra process, which is submitting
```

Comment:
this copied-over comment is a little confusing, now that we're not in the context of main() ?
it'd be clearer if it aligned with the comment around the caller - that this is being called once zebra has requested a bunch of data from the OS or dataplane?

Reply:
fixed as per slack conversation